File: A2/Sem4/MPP/Teme/BileteJavaSocketsORM/BileteServices/src/main/java/bilete/services/BileteException.java
package bilete.services;

public class BileteException extends Exception{
    public BileteException() {
    }

    public BileteException(String message) {
        super(message);
    }

    public BileteException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: A2/Sem4/MPP/Teme/TemaCsNetworking/services/BileteException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace bilete.services
{
    public class BileteException : Exception
    {
        public BileteException() : base() { }

        public BileteException(string message) : base(message) { }

        public BileteException(string message, Exception innerException) : base(message, innerException) { }
    }
}
